src/components/views/PlayWhites.js

```diff
@@ -128,15 +128,32 @@ const PlayWhites = () => {
 
         // white cards get displayed if fetched and a blackCard is not null
         if(cards) {
+            const cardsOnHand = cards.slice(0, cardsPerHand);
             cardsContent =
                 <ul className={"game card-list"}>
-                    {cards.map(card => (
+                    {cardsOnHand.map(card => (
                         <WhiteCard card={card} key={card.id}/>
                     ))}
                 </ul>
         }
     }
 
+    let drawtext = "Somehow there don't seem to be any cards today"
```

Comment:
Can this ever be the case? My assumption would be that there is some bug, maybe just tell the user that there are no more cards left to play today.

Reply:
This can be the case if you already played all white cards go back to the main menu and then go back to the playWhites